src/main/apiClient.ts

```diff
@@ -917,8 +915,8 @@ export default class ApiClient {
           ...authParams,
           ...challenge
         },
-        withCookies: params.method === 'session',
-        accessToken: (params.method === 'access_token') ? params.accessToken : undefined
+        withCookies: params.accessToken === undefined,
+        accessToken: (params.accessToken !== undefined) ? params.accessToken : undefined
```

Comment:
This seems to just be `accessToken: params.accessToken`.

Reply:
It is indeed